caas/kubernetes/provider/resources/applier.go

```diff
@@ -90,3 +90,7 @@ func (a *applier) Run(ctx context.Context, client kubernetes.Interface, noRollba
 	}
 	return nil
 }
+
+func int64Ptr(v int64) *int64 {
```

Comment:
This should go in utils if one doesn't already exist ?

Reply:
there may even be a helper in the sdk itself. the aws and azure sdks have such helpers